### ISSUE TITLE
[Refactor] Updating to Accommodate StreamIO Changes

### ISSUE THREAD

**Issue by sejli** [opensearch-project/search-processor#167]
### Description
Following [this refactoring](https://github.com/opensearch-project/OpenSearch/pull/8157) of the core repo, `search-processor` no longer builds when using OpenSearch as a dependency. The corresponding imports need to be updated to match the current state of OpenSearch.

### Sample Output
Building with `./gradlew build` after the aforementioned change results in the following output:
```
search-processor (721a695) ✔ ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-8.1-bin.zip
...........10%............20%............30%............40%............50%............60%............70%............80%...........90%............100%

Welcome to Gradle 8.1!

Here are the highlights of this release:
 - Stable configuration cache
 - Experimental Kotlin DSL assignment syntax
 - Building with Java 20

For more details see https://docs.gradle.org/8.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 8.1
  OS Info               : Mac OS X 12.6.6 (aarch64)
  JDK Version           : 11 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/jdk-11.0.17+8/Contents/Home
  Random Testing Seed   : 44A9031C4B4DFF28
  In FIPS 140 mode      : false
=======================================

> Task :compileJava
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/ResultTransformerConfigurationFactory.java:10: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/TransformerConfiguration.java:13: error: cannot find symbol
import org.opensearch.common.io.stream.Writeable;
                                      ^
  symbol:   class Writeable
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/TransformerConfiguration.java:17: error: cannot find symbol
public abstract class TransformerConfiguration implements Writeable, ToXContentObject {
                                                          ^
  symbol: class Writeable
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/ResultTransformerConfigurationFactory.java:40: error: cannot find symbol
    ResultTransformerConfiguration configure(StreamInput streamInput) throws IOException;
                                             ^
  symbol:   class StreamInput
  location: interface ResultTransformerConfigurationFactory
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:10: error: cannot find symbol
import org.opensearch.common.ParsingException;
                            ^
  symbol:   class ParsingException
  location: package org.opensearch.common
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:11: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:12: error: cannot find symbol
import org.opensearch.common.io.stream.StreamOutput;
                                      ^
  symbol:   class StreamOutput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:38: error: cannot find symbol
    public SearchConfigurationExtBuilder(StreamInput input, Map<String, ResultTransformerConfigurationFactory> resultTransformerMap) throws IOException {
                                         ^
  symbol:   class StreamInput
  location: class SearchConfigurationExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:51: error: cannot find symbol
    public void writeTo(StreamOutput out) throws IOException {
                        ^
  symbol:   class StreamOutput
  location: class SearchConfigurationExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankerSettings.java:15: error: cannot find symbol
import org.opensearch.common.settings.SecureString;
                                     ^
  symbol:   class SecureString
  location: package org.opensearch.common.settings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/client/KendraClientSettings.java:24: error: cannot find symbol
import org.opensearch.common.settings.SecureString;
                                     ^
  symbol:   class SecureString
  location: package org.opensearch.common.settings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankerSettings.java:98: error: cannot find symbol
  public static final Setting<SecureString> ACCESS_KEY_SETTING = SecureSetting.secureString("kendra_intelligent_ranking.aws.access_key", null);
                              ^
  symbol:   class SecureString
  location: class KendraIntelligentRankerSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankerSettings.java:103: error: cannot find symbol
  public static final Setting<SecureString> SECRET_KEY_SETTING = SecureSetting.secureString("kendra_intelligent_ranking.aws.secret_key", null);
                              ^
  symbol:   class SecureString
  location: class KendraIntelligentRankerSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankerSettings.java:108: error: cannot find symbol
  public static final Setting<SecureString> SESSION_TOKEN_SETTING = SecureSetting.secureString("kendra_intelligent_ranking.aws.session_token", null);
                              ^
  symbol:   class SecureString
  location: class KendraIntelligentRankerSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationFactory.java:10: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationFactory.java:42: error: cannot find symbol
    public ResultTransformerConfiguration configure(StreamInput streamInput) throws IOException {
                                                    ^
  symbol:   class StreamInput
  location: class KendraIntelligentRankingConfigurationFactory
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:10: error: cannot find symbol
import org.opensearch.common.ParsingException;
                            ^
  symbol:   class ParsingException
  location: package org.opensearch.common
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:11: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:12: error: cannot find symbol
import org.opensearch.common.io.stream.StreamOutput;
                                      ^
  symbol:   class StreamOutput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:13: error: cannot find symbol
import org.opensearch.common.io.stream.Writeable;
                                      ^
  symbol:   class Writeable
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:125: error: cannot find symbol
  public static class KendraIntelligentRankingProperties implements Writeable, ToXContentObject {
                                                                    ^
  symbol:   class Writeable
  location: class KendraIntelligentRankingConfiguration
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:56: error: cannot find symbol
  public KendraIntelligentRankingConfiguration(StreamInput input) throws IOException {
                                               ^
  symbol:   class StreamInput
  location: class KendraIntelligentRankingConfiguration
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:75: error: cannot find symbol
  public void writeTo(StreamOutput out) throws IOException {
                      ^
  symbol:   class StreamOutput
  location: class KendraIntelligentRankingConfiguration
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:156: error: cannot find symbol
    public KendraIntelligentRankingProperties(StreamInput input) throws IOException {
                                              ^
  symbol:   class StreamInput
  location: class KendraIntelligentRankingProperties
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:163: error: cannot find symbol
    public void writeTo(StreamOutput out) throws IOException {
                        ^
  symbol:   class StreamOutput
  location: class KendraIntelligentRankingProperties
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/model/KendraIntelligentRankingException.java:12: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/model/KendraIntelligentRankingException.java:15: error: cannot find symbol
  public KendraIntelligentRankingException(StreamInput in) throws IOException {
                                           ^
  symbol:   class StreamInput
  location: class KendraIntelligentRankingException
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:10: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:11: error: cannot find symbol
import org.opensearch.common.io.stream.StreamOutput;
                                      ^
  symbol:   class StreamOutput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:12: error: cannot find symbol
import org.opensearch.common.io.stream.Writeable;
                                      ^
  symbol:   class Writeable
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:23: error: cannot find symbol
public class PersonalizeRequestParameters implements Writeable, ToXContentObject {
                                                     ^
  symbol: class Writeable
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:56: error: cannot find symbol
    public PersonalizeRequestParameters(StreamInput input) throws IOException {
                                        ^
  symbol:   class StreamInput
  location: class PersonalizeRequestParameters
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:78: error: cannot find symbol
    public void writeTo(StreamOutput out) throws IOException {
                        ^
  symbol:   class StreamOutput
  location: class PersonalizeRequestParameters
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParametersExtBuilder.java:12: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParametersExtBuilder.java:13: error: cannot find symbol
import org.opensearch.common.io.stream.StreamOutput;
                                      ^
  symbol:   class StreamOutput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParametersExtBuilder.java:30: error: cannot find symbol
    public PersonalizeRequestParametersExtBuilder(StreamInput input) throws IOException {
                                                  ^
  symbol:   class StreamInput
  location: class PersonalizeRequestParametersExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParametersExtBuilder.java:65: error: cannot find symbol
    public void writeTo(StreamOutput out) throws IOException {
                        ^
  symbol:   class StreamOutput
  location: class PersonalizeRequestParametersExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:15: error: cannot find symbol
import org.opensearch.common.settings.SecureString;
                                     ^
  symbol:   class SecureString
  location: package org.opensearch.common.settings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:31: error: cannot find symbol
    public static final Setting<SecureString> ACCESS_KEY_SETTING = SecureSetting.secureString("personalized_search_ranking.aws.access_key", null);
                                ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:36: error: cannot find symbol
    public static final Setting<SecureString> SECRET_KEY_SETTING = SecureSetting.secureString("personalized_search_ranking.aws.secret_key", null);
                                ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:41: error: cannot find symbol
    public static final Setting<SecureString> SESSION_TOKEN_SETTING = SecureSetting.secureString("personalized_search_ranking.aws.session_token", null);
                                ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:23: error: cannot find symbol
import org.opensearch.common.io.stream.NamedWriteableAwareStreamInput;
                                      ^
  symbol:   class NamedWriteableAwareStreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:24: error: cannot find symbol
import org.opensearch.common.io.stream.NamedWriteableRegistry;
                                      ^
  symbol:   class NamedWriteableRegistry
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:25: error: cannot find symbol
import org.opensearch.common.io.stream.StreamInput;
                                      ^
  symbol:   class StreamInput
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:56: error: cannot find symbol
    private final NamedWriteableRegistry namedWriteableRegistry;
                  ^
  symbol:   class NamedWriteableRegistry
  location: class SearchActionFilter
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/SearchRelevancePlugin.java:14: error: cannot find symbol
import org.opensearch.common.io.stream.NamedWriteableRegistry;
                                      ^
  symbol:   class NamedWriteableRegistry
  location: package org.opensearch.common.io.stream
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/SearchRelevancePlugin.java:92: error: cannot find symbol
            NamedWriteableRegistry namedWriteableRegistry,
            ^
  symbol:   class NamedWriteableRegistry
  location: class SearchRelevancePlugin
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/ResultTransformerConfigurationFactory.java:24: error: name clash: interface ResultTransformerConfigurationFactory has two methods with the same erasure, yet neither overrides the other
    ResultTransformerConfiguration configure(Settings indexSettings);
                                   ^
  first method:  configure(StreamInput) in ResultTransformerConfigurationFactory
  second method: configure(XContentParser) in ResultTransformerConfigurationFactory
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:70: error: cannot find symbol
            throw new ParsingException(
                      ^
  symbol:   class ParsingException
  location: class SearchConfigurationExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java:100: error: cannot find symbol
                throw new ParsingException(
                          ^
  symbol:   class ParsingException
  location: class SearchConfigurationExtBuilder
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/client/KendraClientSettings.java:73: error: cannot find symbol
    try (SecureString key = ACCESS_KEY_SETTING.get(settings);
         ^
  symbol:   class SecureString
  location: class KendraClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/client/KendraClientSettings.java:74: error: cannot find symbol
        SecureString secret = SECRET_KEY_SETTING.get(settings);
        ^
  symbol:   class SecureString
  location: class KendraClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/client/KendraClientSettings.java:75: error: cannot find symbol
        SecureString sessionToken = SESSION_TOKEN_SETTING.get(settings)) {
        ^
  symbol:   class SecureString
  location: class KendraClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationFactory.java:42: error: configure(StreamInput) in KendraIntelligentRankingConfigurationFactory cannot implement configure(Settings) in ResultTransformerConfigurationFactory
    public ResultTransformerConfiguration configure(StreamInput streamInput) throws IOException {
                                          ^
  overridden method does not throw IOException
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationFactory.java:32: error: name clash: class KendraIntelligentRankingConfigurationFactory has two methods with the same erasure, yet neither overrides the other
    public ResultTransformerConfiguration configure(Settings indexSettings) {
                                          ^
  first method:  configure(StreamInput) in KendraIntelligentRankingConfigurationFactory
  second method: configure(XContentParser) in KendraIntelligentRankingConfigurationFactory
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationFactory.java:37: error: name clash: class KendraIntelligentRankingConfigurationFactory has two methods with the same erasure, yet neither overrides the other
    public ResultTransformerConfiguration configure(XContentParser parser) throws IOException {
                                          ^
  first method:  configure(StreamInput) in KendraIntelligentRankingConfigurationFactory
  second method: configure(Settings) in ResultTransformerConfigurationFactory
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:74: error: method does not override or implement a method from a supertype
  @Override
  ^
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:84: error: cannot find symbol
        throw new ParsingException(parser.getTokenLocation(),
                  ^
  symbol:   class ParsingException
  location: class KendraIntelligentRankingConfiguration
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:89: error: cannot find symbol
      throw new ParsingException(parser.getTokenLocation(), iae.getMessage(), iae);
                ^
  symbol:   class ParsingException
  location: class KendraIntelligentRankingConfiguration
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:162: error: method does not override or implement a method from a supertype
    @Override
    ^
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfiguration.java:179: error: cannot find symbol
        throw new ParsingException(parser.getTokenLocation(), iae.getMessage(), iae);
                  ^
  symbol:   class ParsingException
  location: class KendraIntelligentRankingProperties
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/requestparameter/PersonalizeRequestParameters.java:77: error: method does not override or implement a method from a supertype
    @Override
    ^
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:59: error: cannot find symbol
        try (SecureString key = ACCESS_KEY_SETTING.get(settings);
             ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:60: error: cannot find symbol
             SecureString secret = SECRET_KEY_SETTING.get(settings);
             ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/client/PersonalizeClientSettings.java:61: error: cannot find symbol
             SecureString sessionToken = SESSION_TOKEN_SETTING.get(settings)) {
             ^
  symbol:   class SecureString
  location: class PersonalizeClientSettings
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:63: error: cannot find symbol
        namedWriteableRegistry = new NamedWriteableRegistry(Collections.emptyList());
                                     ^
  symbol:   class NamedWriteableRegistry
  location: class SearchActionFilter
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:211: error: cannot find symbol
                    final StreamInput in = new NamedWriteableAwareStreamInput(out.bytes().streamInput(),
                          ^
  symbol: class StreamInput
/Users/lnse/opensearch/search-processor/src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java:211: error: cannot find symbol
                    final StreamInput in = new NamedWriteableAwareStreamInput(out.bytes().streamInput(),
                                               ^
  symbol: class NamedWriteableAwareStreamInput
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
68 errors

> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 28s
2 actionable tasks: 2 executed
```

**Comment by sejli** [comment]
@kulket I have updated the imports and merged the changes in #168. I saw that you are making changes to the plugin, just wanted to let you know to rebase before making any changes!

**Comment by sejli** [comment]
#170 has been merged in, closing this issue.
